skynet/constants.py

```diff
@@ -1,113 +1,115 @@
-#!/usr/bin/python
 import msgspec
+
+from enum import Enum
 from typing import Literal
 
 VERSION = '0.1a12'
 
 DOCKER_RUNTIME_CUDA = 'skynet:runtime-cuda'
 
 
-class Size(msgspec.Struct):
-    w: int
-    h: int
-
-
 class ModelDesc(msgspec.Struct):
-    short: str
-    mem: float
-    size: Size
-    tags: list[Literal['txt2img', 'img2img', 'inpaint']]
+    short: str  # short unique name
+    mem: float  # recomended mem
+    attrs: dict  # additional mode specific attrs
+    tags: list[Literal['txt2img', 'img2img', 'inpaint', 'upscale']]
 
 
 MODELS: dict[str, ModelDesc] = {
+    'RealESRGAN_x4plus': ModelDesc(
+        short='realesrgan',
+        mem=4,
```

Comment:
what unit tho?  a comment on the def for this would be handy.

Reply:
Its in GB, Adding comment in `structify` branch